### DISCUSSION TITLE
Install specific cask to external ssd

### DISCUSSION THREAD

**Discussion by LoCrealloc** [Homebrew/discussions#122] [Everyday usage] [answered]
Hey, I want homebrew to install a specific cask to an external ssd, but not all casks! Is there any way to do so? Thanks for your replies!!!

**Comment by SMillerDev** [comment, marked as answer]
Yes, [https://github.com/Homebrew/homebrew-cask/blob/master/USAGE.md#options](https://github.com/Homebrew/homebrew-cask/blob/master/USAGE.md#options) see `--appdir`

**Reply by LoCrealloc** [reply]
Thank you very much, I actually dont know why I didnt find it myself...
